frontend/shared/textCleanup/App.js

```diff
@@ -86,15 +88,31 @@ class App extends Component {
                 </p>
                 <b>To begin, select a data-type to cleanup</b>
                 <ul>
-                    {store.assessmentMetadata.items
-                        .filter(d => d.count > 0)
-                        .map(d => {
+                    {Object.entries(groupedItems)
+                        .filter(([app, items]) => items.some(item => item.count > 0))
+                        .map(([app, items]) => {
                             return (
-                                <li key={d.type}>
-                                    <a href="#" onClick={() => store.selectModel(d)}>
-                                        {d.count}&nbsp;{d.title}
-                                    </a>
-                                </li>
+                                <div key={app}>
+                                    <h4 className="mt-3">{app}:</h4>
+                                    <ul className="list-group">
+                                        {items
+                                            .filter(item => item.count > 0)
+                                            .map(d => {
+                                                return (
```

Comment:
I like this change, but I think it's breaking our integration test. To fix, you'll have to run the integration tests locally. I think you can do something like this:

```bash
playwright install --with-deps chromium
set INTEGRATION_TESTS=1
set PWDEBUG=0
py.test -sv tests/integration/test_cleanup.py
```

Then you can record the new thing; or just hand-edit the text to make edits. I don't really worry about re-recording the whole test; just the thing that failed.  In this case, I think the "text" is different than what it was before, since you switched the count to a badge

Adapted from https://github.com/shapiromatron/hawc/blob/aae68b1cf6fb746efeaf0a57a9611862cc27ae62/make.bat#L102-L107